fe/fe-core/src/main/java/org/apache/doris/rewrite/FEFunctions.java

```diff
@@ -231,21 +231,21 @@ public static IntLiteral unixTimestamp2(LiteralExpr arg) throws AnalysisExceptio
         return new IntLiteral(unixTime, Type.INT);
     }
 
-    @FEFunction(name = "from_unixtime", argTypes = { "INT" }, returnType = "VARCHAR")
+    @FEFunction(name = "from_unixtime", argTypes = { "BIGINT" }, returnType = "VARCHAR")
     public static StringLiteral fromUnixTime(LiteralExpr unixTime) throws AnalysisException {
         // if unixTime < 0, we should return null, throw a exception and let BE process
-        if (unixTime.getLongValue() < 0 || unixTime.getLongValue() >= Integer.MAX_VALUE) {
+        if (unixTime.getLongValue() < 0 || unixTime.getLongValue() > 32536771199L) {
```

Comment:
pls add some comment about the magic number 32536771199L. And do same in nereids

Reply:
done